packages/extension/src/index.ts

```diff
@@ -22,9 +22,19 @@ export async function activate (context: vscode.ExtensionContext) {
      * demo code for example widget in "/example"
      */
     marquee: {
-      setup: (tangle: Client<{ counter: number }>) => {
+      customWidgetCounter: 2,
+      setup: (tangle: Client<{ counter: number, changeName: string }>) => {
         return tangle.whenReady().then(() => {
           let i = 0
+          let name = 'foo'
+          if (true){
+            tangle.emit('changeName', name)
+          }
+
+          tangle.on('changeName', (newName) => {
+            name = newName
+          })
```

Comment:
I don't quite get what this does? `if (true)` looks suspicious, why do we need the if here when it will be always `true`?